src/lib/instrument-listen.js

```diff
@@ -12,14 +12,14 @@ const fs = require('fs');
 const readFile = denodeify(fs.readFile);
 
 module.exports = (app, meta, initPromises) => {
-	async function createServer(app) {
+	async function createServer (app) {
```

Comment:
**nitpick**: there's a lot of unrelated formatting changes, i guess they've come from eslint running on commit? if they have come from eslint, i'd prefer them to be split into a separate PR

Reply:
oo I missed that (clearly not got my head in the game yet today), yes let me undo those....